packages/components/src/ui/base-button/component.js

```diff
@@ -44,7 +44,7 @@ function BaseButton( props, forwardedRef ) {
 		isFocused = false,
 		isLoading = false,
 		noWrap = true,
-		pre,
+		prefix,
```

Comment:
@ItsJonQ Unfortunately it's not possible for this prop to be named `prefix` because it conflicts with the HTMLAttribute of the same name.

I'm not sure why there's an issue with this but currently the only workaround I could think of was to rename it.

Once the style system is merged into G2 we can try to fix this more easily. Currently it's very difficult to tinker with things to try to make them work.

Reply:
Whoops! I totally missed this 🙏 

That's a bit of a bummer. Will change it back 🙈